docs/en/enterprise-edition/content-collections/application-security/get-started/connect-code-and-build-providers/ci-cd-runs/add-checkov.adoc

```diff
@@ -13,7 +13,11 @@ See https://www.checkov.io/2.Basics/Visualizing%20Checkov%20Output.html[Visualiz
 . Before you begin.
 .. xref:../../../../administration/create-access-keys.adoc[Generate and copy the Prisma Cloud access key] to enable access to Prisma Cloud. The access key includes a key ID and secret.
 .. Add the Prisma Cloud IP addresses and hostname for Application Security to an xref:../../../../get-started/console-prerequisites.adoc[allow list] to enable access to the Prisma Cloud Console. 
-.. Grant *Administrator* permissions in the relevant organization to the Prisma user installing Checkov.
+.. Grant either the *Developer*, *AppSec Admin* or *System Admin* permissions in the relevant organization to the Prisma user installing Checkov. If you prefer to use a custom permission group, Checkov requires:
```

Comment:
Grant the user installing Checkov either the Developer, AppSec Admin, or System Admin role within Prisma.